ingest/user_de.py

```diff
@@ -170,15 +177,31 @@ def generate_individual_files(col, genes, rest, groups, clean_val, qual):
 
 # final result: individual files for each comparison
 
+def generate_manifest(clean_val, qual):
```

Comment:
Blocker: I believe what we had discussed was a tab-separated file where each "group" observation was a separate _line_, not a _column_.  This will make processing it much simpler back in the portal.  So the file should read:
```
type_0
type_1
...
```

Also, if we use that format, we don't need two manifests.  We can have a single two-column manifest where a line with one value is considered one vs. rest, and a line with two values is pairwise.  E.g:
```
type_0
type_1
type_2
type_0  type_1
type_0  type_2
type_1  type_2
```